improver_tests/wind_calculations/wind_downscaling/test_RoughnessCorrection.py

```diff
@@ -31,140 +31,83 @@
 """Unit tests for plugin wind_downscaling.RoughnessCorrection."""
 
 
+import datetime
 import unittest
 
 import iris
 import numpy as np
 from cf_units import Unit
-from iris.coords import AuxCoord
 from iris.tests import IrisTest
 
 from improver.constants import RMDI
-from improver.grids import STANDARD_GRID_CCRS
+from improver.synthetic_data.set_up_test_cubes import (
+    add_coordinate,
+    set_up_variable_cube,
+)
 from improver.wind_calculations.wind_downscaling import RoughnessCorrection
 
 
-def _make_ukvx_grid():
-    """
-    Create a two-dimensional Cube that represents the UK model standard
-    grid (UKVX).
-
-    Returns
-    -------
-    iris.cube.Cube instance
-        Cube mapped to the UKVX grid.
-
-    """
+def _make_flat_cube(data, name, unit):
+    """Create a "flat" y/x cube with required shape and 2 km grid bounds"""
+    flat_cube = set_up_variable_cube(
+        data,
+        name=name,
+        units=unit,
+        spatial_grid="equalarea",
+        domain_corner=(-1036000, -1158000),
+        grid_spacing=2000,
+    )
+    for axis in ["x", "y"]:
+        points = flat_cube.coord(axis=axis).points
+        flat_cube.coord(axis=axis).bounds = np.array(
+            [points - 1000.0, points + 1000.0]
+        ).T
+    return flat_cube
+
+
+def make_ancil_cube(data, name, unit, shape=None):
+    """Create an ancillary cube (constant with time)"""
+    data = np.array(data, dtype=np.float32)
+    if shape is not None:
+        data = data.reshape(shape)
+    cube = _make_flat_cube(data, name, unit)
+    for coord in ["time", "forecast_reference_time", "forecast_period"]:
+        cube.remove_coord(coord)
+    return cube
 
-    # Grid resolution
-    num_x, num_y = 1042, 970
 
-    data = np.zeros([num_y, num_x])
+def make_point_height_cube(heights_data):
+    """Create a multi-level height ancillary for one spatial point"""
+    flat_cube = make_ancil_cube(1, None, None, shape=(1, 1))
+    cube = add_coordinate(flat_cube, range(len(heights_data)), "model_level_number", 1)
+    cube.data = np.array(heights_data).reshape(len(heights_data), 1, 1)
```

Comment:
Codacy found an issue: [Too many positional arguments for method call](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=6350585)